source/mobile/mobile-overview.rst

```diff
@@ -32,3 +32,8 @@ To meet other security and compliance policies, you can :doc:`deploy the Matterm
 This option is recommended for:
 
 1. Organizations that typically use EMM solutions to deploy mobile apps to meet security and compliance policies
+
+EMM vs. App Wrapping
+--------------------
+
+Using an app wrapper, like the `Citrix MDX Toolkit <https://docs.citrix.com/en-us/mdx-toolkit/about-mdx-toolkit.html>`_ is *not supported by Mattermost* because it injects code into the app that we cannot control. Many app wrapping toolkits also provide EMM, so contact your provider for more information about how to deploy Mattermost with EMM.
```

Comment:
@esethna - can you review the line on why we don't support it? I think it's untested and some EMM solutions may require changes to the app to make wrapping work. 

Also "EMM" is just a general term and applies to both methods of deploying, so we should update to App Wrapping vs Managed App Config and maybe add some explanation on what each is. 

Citrix is an example of an EMM provider, and I believe they have both [app wrapping](https://docs.citrix.com/en-us/mdx-toolkit/about-mdx-toolkit.html) and [managed app config](https://docs.citrix.com/en-us/xenmobile/server/policies/app-configuration-policy.html).

Reply:
Updated and committed to this branch if you want to review.